scout/index.md

```diff
@@ -71,3 +71,19 @@ format. You can print the results directly to stdout, or export them to a file
 using a structured format, such as Static Analysis Results Interchange Format
 (SARIF). For more information about how to use the `docker scout` CLI, see the
 [reference documentation](../engine/reference/commandline/scout_cves.md).
+
+The plugin is available in Docker Desktop starting with version 4.17 and available
+as a standalone binary.
+
+To install the plugin, run the following command:
+
+```bash
+curl -sSfL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh | sh -s --
+```
+
+> **Tip**
+>
+> If you want to install the plugin manually, you can find full instructions in the plugin's repository.
```

Comment:
Could we add a link to the repository here?

```suggestion
> If you want to install the plugin manually, you can find full instructions in the [plugin's repository](https://github.com/docker/scout-cli).
```

Reply:
Hmm, odd, I did have some git merge issues when trying to squash.